testsuite/features/support/twopence_init.rb

```diff
@@ -131,6 +131,8 @@ def twopence_init(target)
   node.init_full_hostname(fqdn)
 
   STDOUT.puts "Host '#{$named_nodes[node.hash]}' is alive with determined hostname #{hostname.strip} and FQDN #{fqdn.strip}" unless $build_validation
+  result, _code = node.run('grep PRETTY /etc/os-release')
+  puts "'#{$named_nodes[node.hash]}' is running OS #{result}"
```

Comment:
You could also have used `node.full_hostname` or `hostname.strip`, but that's probably ok that way too

In the past, `STDOUT.puts` was needed to work around jenkin's buffering, but maybe it's not needed anymore

Reply:
Switched `puts` with `STDOUT.puts` just in case and to be consistent with its use two lines above.